juntagrico/config.py

```diff
@@ -69,6 +69,7 @@ class Config:
     extra_sub_info = _get_setting('EXTRA_SUB_INFO')
     activity_area_info = _get_setting('ACTIVITY_AREA_INFO')
     enable_shares = _get_setting('ENABLE_SHARES', True)
+    enable_share_certificates = _get_setting('ENABLE_SHARE_CERTIFICATES', False)
```

Comment:
würde es nicht mehr sinn machen das aus enable_shares und der lokale zusammenzurechenen (falls eneable shares aktiviert ist und wir ch in der lokale haben soll das angezeigt werden)?

Reply:
Grundsätzlich kann es ja auch in anderen Ländern gewünscht sein. Wollte es nur nicht per default aktivieren, sondern die Wahl lassen. Einverstanden?

Reply:
andere länder andere sitten, dh das muss in anderen ländern eventuell anders aussehen.
zusätzlich sehe ich keinen grund wieso man das abschalten müsste